test/functional/ui/statuscolumn_spec.lua

```diff
@@ -713,6 +744,40 @@ describe('statuscolumn', function()
                                                                |
         ]])
       end)
+
+      it('foldcolumn item can be clicked', function()
+        api.nvim_set_option_value('statuscolumn', '|%C|', {})
+        api.nvim_set_option_value('foldcolumn', '2', {})
+        feed('ggzfjzfjzo')
+        local s1 = [[
+          {8:|}{7:-+}{8:|}{13:^+---  2 lines: aaaaa·····························}|
+          {8:|}{7:│ }{8:|}aaaaa                                            |
+          {8:|}{7:  }{8:|}aaaaa                                            |*11
+                                                               |
+        ]]
+        screen:expect(s1)
+        api.nvim_input_mouse('left', 'press', '', 0, 0, 2)
+        screen:expect([[
+          {8:|}{7:--}{8:|}^aaaaa                                            |
+          {8:|}{7:││}{8:|}aaaaa                                            |
+          {8:|}{7:│ }{8:|}aaaaa                                            |
+          {8:|}{7:  }{8:|}aaaaa                                            |*10
+                                                               |
+        ]])
+        -- Note: alternate between left and right mouse buttons to avoid triggering multiclicks
+        api.nvim_input_mouse('right', 'press', '', 0, 0, 2)
+        feed('<Esc>') -- Close popupmenu
+        api.nvim_input_mouse('left', 'press', '', 0, 0, 1)
+        screen:expect(s1)
+        api.nvim_input_mouse('right', 'press', '', 0, 0, 2)
+        feed('<Esc>') -- Close popupmenu
```

Comment:
Adding right clicks for the sole purpose of avoid multiclicks is a bit awkward. How about setting `'mousetime'` to zero instead?

Reply:
I copied this from a test in `ui/statusline_spec.lua` (which you originally added), but if that works sure.